resolver/src/test/java/io/netty/resolver/DefaultHostsFileEntriesResolverTest.java

```diff
@@ -28,6 +32,21 @@
     @Test
     public void testLocalhost() {
         DefaultHostsFileEntriesResolver resolver = new DefaultHostsFileEntriesResolver();
+        //make sure there's _some_ localhost entry. put one if there isnt
```

Comment:
@radai-rosenblatt after thinking more about it maybe we should just use `Assume.*` for this test and revert the rest ?